.travis.yml

```diff
@@ -11,6 +11,7 @@ rvm:
   - 2.0
   - 2.1
   - 2.2
+  - 2.3
```

Comment:
Looks like this shortened version isn't supported on Travis, at least as of the time this got pushed.